<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testBase</class>
 <widget class="QWidget" name="testBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testBase</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
        background-color: #2E2E2E;
        font-family: Arial, sans-serif;
        }

        QPushButton {
        border: 2px solid #444;
        border-radius: 5px;
        background-color: #333;
        color: #FFFFFF;
        padding: 8px 15px;
        margin: 5px;
        }

        QPushButton:hover {
        background-color: #FF5733;
        }

        QPushButton:pressed {
        background-color: #C70039;
        }

        QLineEdit {
        border: 2px solid #444;
        border-radius: 5px;
        padding: 8px;
        background-color: #1C1C1C;
        color: #FFFFFF;
        margin-right: 10px;
        }

        QLineEdit:focus {
        border-color: #FF5733;
        background-color: #333;
        }

        QCalendarWidget QToolButton {
        height: 30px;
        width: 70px;
        color: #FFFFFF;
        font-size: 14px;
        icon-size: 20px, 20px;
        background-color: #FF5733;
        border: none;
        margin: 5px;
        }

        QCalendarWidget QToolButton:hover {
        background-color: #C70039;
        }

        QCalendarWidget QMenu {
        width: 100px;
        background-color: #333;
        color: #FFFFFF;
        }

        QCalendarWidget QSpinBox {
        width: 50px;
        font-size: 14px;
        color: #FFFFFF;
        background-color: #1C1C1C;
        selection-background-color: #FF5733;
        selection-color: #000000;
        }

        QCalendarWidget QWidget#qt_calendar_navigationbar {
        background-color: #333;
        }

        QCalendarWidget QTableView {
        border-width: 1px;
        border-color: #444;
        border-style: solid;
        border-radius: 5px;
        gridline-color: #444;
        background-color: #1C1C1C;
        }
            QCalendarWidget QHeaderView::section {
            background-color: #333;
            color: #FFFFFF;
            padding: 5px;
            border: 1px solid #444;
            }</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ТЕСТ</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>891</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="questionLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 14px</string>
      </property>
      <property name="text">
       <string>Как определить базовый конструктор c++</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="answerLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>901</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="previousButton">
      <property name="text">
       <string>НАЗАД</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>ЗАВЕРШИТЬ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>ДАЛЕЕ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
